tfhe/src/core_crypto/gpu/vec.rs

```diff
@@ -345,6 +345,20 @@ impl<T: Numeric> CudaVec<T> {
         }
     }
 
+    /// Returns a copy of the vector
+    ///
+    /// # Safety
+    ///
+    /// - [CudaStreams::synchronize] __must__ be called as soon as synchronization is required
```

Comment:
This function should not be unsafe and the comment above can be removed.

Reply:
Fixed!